src/Classes/DocumentUpdater.php

```diff
@@ -18,11 +18,13 @@ class DocumentUpdater
 
   protected $storePath;
   protected $primaryKey;
+  protected $prettyPrint = false;
 
-  public function __construct(string $storePath, string $primaryKey)
+  public function __construct(string $storePath, string $primaryKey, bool $prettyPrint)
```

Comment:
Type declarations are supported in PHP >= 7.4

Reply:
See https://github.com/rakibtg/SleekDB/pull/234#discussion_r878758462